### PR TITLE
DATAUP-642 advanced parameters errors label

### PR DESCRIPTION
**Pull request by briehl**
# Description of PR purpose/changes

Show a label for errors in the advanced parameters area when hidden.

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/DATAUP-642
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook
- [x] Any dependent changes have been merged and published in downstream modules

# Updating Version and Release Notes (if applicable)

- [n/a] [Version has been bumped](https://semver.org/) for each release
- [x] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js [hunk 1]
```diff
@@ -116,10 +118,14 @@ define([
                 });
 
                 fieldWidget.bus.on('validation', (message) => {
+                    // console.log(parameterSpec.id + ': ' + JSON.stringify(message));
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js:121]: comment left in

File: kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js [hunk 2]
```diff
@@ -423,6 +460,7 @@ define([
             }
             return controlPromise
                 .then((inputWidget) => {
+                    // console.log(`starting ${paramSpec.id} with ${initialParams[paramSpec.id]}`)
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/paramsWidget.js:463]: comment

File: test/unit/spec/common/cellComponents/paramsWidget-Spec.js [hunk 3]
```diff
@@ -67,8 +68,15 @@ define([
             this.paramIds = this.model.getItem(['app', 'otherParamIds', 'fastq_reads']);
 
             this.parameters = this.spec.getSpec().parameters;
+            this.advancedParamIds = [];
+            for (const [, entry] of Object.entries(this.parameters.specs)) {
```
Comment (ialarmedalien) [inline test/unit/spec/common/cellComponents/paramsWidget-Spec.js:72]: or just use `Object.values(this.parameters.specs).forEach((entry) => { ... })`

**Review by ialarmedalien — APPROVED** [review]
Looks like a couple of comments got left in -- looks fine otherwise.

**Comment by briehl** [comment]
I guess there was some slop right before I went on leave. Fixed!

**Pull request merged**